index.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
+require('rootpath')()
```

Comment:
Not if we want to specify paths for ```require```s consistently throughout core.
The current way to specify a path is: ```dirname/filename``` (dirname doesn't start with ```./```) 

I find it much nicer to write ```foo/bar``` instead of ```'./foo/bar'```.